src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/model/OffenderBooking.java

```diff
@@ -108,10 +107,8 @@ public class OffenderBooking extends AuditableEntity {
     @OneToMany(mappedBy = "offenderBooking", cascade = CascadeType.ALL)
     @Default
     @Exclude
-    @BatchSize(size = 25)
     private List<CourtOrder> courtOrders = new ArrayList<>();
 
-    @ListIndexBase(1)
```

Comment:
this annotation looks like it was left on after a copy from a different field.  Doesn't make any sense without an order by but there isn't a sequence on this field anyway and therefore hibernate generated invalid sql.